src/database/schemas/task.ts

```diff
@@ -0,0 +1,43 @@
+import mongoose from 'mongoose';
+import ITask from '../../models/task';
+
+const taskSchema = new mongoose.Schema<ITask>({
+  title: {
+    required: true,
+    type: String
+  },
+  status: {
+    required: true,
+    type: Number
+  },
+  createdAt: {
+    required: true,
+    type: Date
+  },
+  sortId: {
```

Comment:
what's that for? Do you need extra prop for sorting?

Reply:
Yes, if user whats to readjust list of task after creation I need to save this order

Reply:
is it `required` then? I guess it might be empty if the default sorting (using dates or any other prop) is used